### PR TITLE
Merge Endpoints support

### PR DESCRIPTION
**Pull request by dongjing-github**

### CODE REVIEW

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>webauthndemo</application>
-  <version>${appengine.app.version}</version>
+  <application>webauthndemo-176619</application>
```
Comment (cpiper) [inline src/main/webapp/WEB-INF/appengine-web.xml]: Remove this

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -16,9 +16,27 @@
...
   <security-constraint>
     <web-resource-collection>
-      <web-resource-name>all</web-resource-name>
+      <web-resource-name>assetlinks</web-resource-name>
+      <url-pattern>/.well-known/*</url-pattern>
+    </web-resource-collection>
+    <!-- OMIT auth-constraint -->
+  </security-constraint>
+
+  <security-constraint>
```
Comment (cpiper) [inline src/main/webapp/WEB-INF/web.xml]: Is this necessary since the wildcard pattern is used below?

**Comment by dongjing-github** [comment]
I signed it!

**Comment by dongjing-github** [comment]
Closing this pull request, as I made many changes to my local branch. Will open another pull request for merge.

**Pull request closed without merging**